include/deal.II/base/vectorization.h

```diff
@@ -224,6 +224,20 @@ class VectorizedArray
   // POD means that there should be no user-defined constructors, destructors
   // and copy functions (the standard is somewhat relaxed in C++2011, though).
 
+  VectorizedArray()
```

Comment:
I would prefer ` = default` here and in the other places below. Then, you don't need to modify the `static_assertion`s below, i.e. the class is still `trivial` and has `standard_layout`, see https://godbolt.org/z/L9Q1hN.

Reply:
You are right. I have updated the code.